app/src/main/java/edu/uc/coffeens/stocktracker/services/RetrofitClientInstance.kt

```diff
@@ -8,15 +8,15 @@ import retrofit2.converter.gson.GsonConverterFactory
  */
 object RetrofitClientInstance {
 
-    private var retrofit: Retrofit? = null;
+    private var retrofit: Retrofit? = null
     private val BASE_URL = "https://gist.githubusercontent.com/"
 
     val retrofitInstance: Retrofit?
         get() {
             //Has this object been created?
             if (retrofit == null) {
                 // Create retrofit instance
-                retrofit = retrofit2.Retrofit.Builder()
+                retrofit = Retrofit.Builder()
```

Comment:
Optimized how the retrofit builder is built. Looks a bit cleaner, and easier to read.